docs/resources/Auto_Moderation.md

```diff
@@ -0,0 +1,214 @@
+> These string values will most likely be changed to integer values before official API launch
+
+| Type             | Value          | Description                                                  |
+| ---------------- | -------------- | ------------------------------------------------------------ |
+| PROFANITY        | PROFANITY      | Words that may be considered forms of swearing or cursing    |
+| SEXUAL_CONTENT   | SEXUAL_CONTENT | Words that refer to sexually explicit behavior or activity   |
+| SLURS            | SLURS          | Personal insults or words that may be considered hate speech |
+
+
+###### Event Types
+
+Indicates in what event context a rule should be checked.
+
+| Type             | Value   | Description                              |
+| ---------------- | ------- | ---------------------------------------- |
+| MESSAGE_SEND     | 1       | when a member sends a message on a guild |
+
+
+### Auto Moderation Action Object
+
+An action which will execute whenever a rule is triggered.
+
+###### Auto Moderation Action Structure
+
+| Field    | Type                                                                                             | Description                                                               |
+| -------- | ------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------- |
+| type     | [action type](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-action-object-action-types)        | the type of action                                                        |
+| metadata | [action metadata](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-action-object-action-metadata) | additional metadata needed during execution for this specific action type |
+
+
+###### Action Types
+
+| Type                  | Value   | Description                                           |
+| --------------------- | ------- | ----------------------------------------------------- |
+| BLOCK_MESSAGE         | 1       | blocks the content of a message according to the rule |
+| SEND_ALERT_MESSAGE    | 2       | logs user content to a specified channel              |
+| TIMEOUT               | 3       | times out user for specified duration                 |
+
+
+###### Action Metadata
+
+Additional data used when an action is executed. Different fields are relevant based on
+value of [action type](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-action-object-action-types).
+
+| Field      | Type       | Associated Action Types | Description                                    | 
+| ---------- | ---------- | ----------------------- | ---------------------------------------------- |
+| channel_id            | snowflake  | SEND_ALERT_MESSAGE      | channel to which user content should be logged              |
+| duration_seconds      | snowflake  | SEND_ALERT_MESSAGE      | timeout duration in seconds                                 |
```

Comment:
Is there a reason why this is a snowflake and not a regular integer?

Reply:
Was a mistake 😅, I've already changed it locally.